dockers/actions-plan-preview/planpreview.go

```diff
@@ -202,3 +223,18 @@ func makeCommentBody(event *githubEvent, r *PlanPreviewResult) string {
 
 	return b.String()
 }
+
+func groupApplicationResults(apps []ApplicationResult) (changes, pipelines, quicks []ApplicationResult) {
+	for _, app := range apps {
+		if !app.NoChange {
+			changes = append(changes, app)
+			continue
+		}
+		if app.SyncStrategy == "PIPELINE" {
```

Comment:
nit
```suggestion
		if app.SyncStrategy == model.SyncStrategy_PIPELINE.String() {
```

Reply:
This actions package is stand alone so it will not use any PipeCD's packages.

Reply:
True